package.json

```diff
@@ -45,29 +45,29 @@
     "!*/__tests__"
   ],
   "dependencies": {
-    "@loopback/boot": "^5.0.9",
-    "@loopback/core": "^4.0.9",
-    "@loopback/rest": "^12.0.9",
+    "@loopback/boot": "^5.0.10",
+    "@loopback/core": "^4.0.10",
+    "@loopback/rest": "^12.0.10",
```

Comment:
These aren't the latest versions @RaghavaroraSF please update this PR.

Reply:
Updated